packager/platform-bundle-bootstrapper.js

```diff
@@ -0,0 +1,9 @@
+if (global.__platformBundles != undefined) {
```

Comment:
We don't need the platform-bundle-bootstrapper. office-addin-node-debugger supports this.